suzieq/poller/worker/services/interfaces.py

```diff
@@ -592,6 +592,11 @@ def fix_nxos_speed(entry):
                 drop_indices.append(entry_idx)
                 continue
 
+            if entry.get('_entryType', '') == "vrf":
+                entry['type'] = 'vrf'
+                entry['mtu'] = 1500
+                entry['vni'] = 0
```

Comment:
Can you add a comment explaining why you are setting these values? If we will have problem with nxos, these lines will be very confusing to me during the debugging phase

Reply:
These values are not provided by NXOS, ever.